capstone/src/main/java/com/google/sps/data/Card.java

```diff
@@ -20,13 +20,18 @@
   private String back;
   private Integer study_set_id; 
 
-  public Card(Integer id, String infront, String inback, Integer study_set_id){
+  public Card(Integer id, String front, String back, Integer study_set_id){
```

Comment:
`study_set_id` and all references to `this.study_set_id` should be camel case: https://google.github.io/styleguide/javaguide.html#s5.2.6-parameter-names